resources/assets/lib/import-shims.coffee

```diff
@@ -0,0 +1,27 @@
+###
+#    Copyright 2015-2017 ppy Pty. Ltd.
+#
+#    This file is part of osu!web. osu!web is distributed with the hope of
+#    attracting more community contributions to the core ecosystem of osu!.
+#
+#    osu!web is free software: you can redistribute it and/or modify
+#    it under the terms of the Affero GNU General Public License version 3
+#    as published by the Free Software Foundation.
+#
+#    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+#    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#    See the GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+
+# Import shim so that globally declared scripts can work without changes.
+
+import { StoreCheckout } from 'store-checkout'
+import Promise from 'promise-polyfill'
+
+# polyfill non-Edge IE
+window.Promise ?= Promise
+
+window.StoreCheckout = StoreCheckout
```

Comment:
`?=`? Not sure if needed (assuming loaded only once) (also applies everywhere else)

Reply:
imported modules should only evaluate once, even if imported multiple times